apps/generator/docs/nunjucka-depreciate.md

```diff
@@ -0,0 +1,120 @@
+--
+title: "Depreciation of nunjucks render engine"
+weight: 170
+---
+
+# Migration Guide from nunjucks render engine to react render engine
+
+## Introduction
+
+AsyncAPI Generator is moving away from Nunjucks templates in favor of React templates. This guide will help you migrate your existing Nunjucks templates to React.
+
+## Step-by-Step Migration Guide
+
+### 1. Update package.json
+
+Change your template configuration in `package.json`:
```

Comment:
and please explain that once deprecation period is ended, and we remove default nunjucks engine, react will become default and this setting will no longer be needed to configure